src/components/MetaInfoPanel.vue

```diff
@@ -0,0 +1,243 @@
+<template>
```

Comment:
This is the new meta debug panel that we show at the top of the page locally! It fetches metadata from the actual page, helping confirm everything is wired up properly